x-pack/test/functional/apps/rollup_job/rollup_jobs.js

```diff
@@ -4,64 +4,70 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+
+import datemath from '@elastic/datemath';
 import expect from '@kbn/expect';
-import { indexBy } from 'lodash';
+import { mockIndices } from './hybrid_index_helper';
 
 export default function ({ getService, getPageObjects }) {
-  const esArchiver = getService('esArchiver');
-  const log = getService('log');
-  const PageObjects = getPageObjects(['security', 'rollup', 'common', 'header']);
+
+  const es = getService('es');
+  const PageObjects = getPageObjects(['security', 'rollup', 'common', 'indexManagement', 'settings', 'discover']);
 
   describe('rollup job', function () {
-    this.tags('smoke');
-    before(async () => {
-      // init data
-      await esArchiver.loadIfNeeded('logstash_functional');
-      await esArchiver.load('canvas/default');
+    const rollupJobName = 'rollup-to-be-' + Math.floor(Math.random() * 10000);
+    const indexName = 'rollup-to-be';
+    //make sure all dates have the same concept of "now"
+    const now = new Date();
+    const pastDates = [
+      datemath.parse('now-1d', { forceNow: now }),
+      datemath.parse('now-2d', { forceNow: now }),
+      datemath.parse('now-3d', { forceNow: now }),
+    ];
+
+    it('create new rollup job', async () => {
```

Comment:
It was my understanding of the feature that part of the success criteria of creating a rollup job was that the rollup index was created. This assertion does not check that the rollup was actually successfully triggered but that the required index was created which is supposed to happen as soon as you click `Create new job` or `Save`. To me, it seemed like a good idea to capture that but I can also see how it could be something that has crossed over into the ES realm rather than the UI. I can revert since this does encroach a bit.

Reply:
Yes, I think we should go back to the original test. I think we should try to constrain our tests to behavior that Kibana is responsible for, so that when a test fails we can easily trace back to the cause and address it.

Reply:
This is fixed.